### ISSUE TITLE
Red cross on white ground with auto hide

### ISSUE THREAD

**Issue by ELMI5** [dockpanelsuite/dockpanelsuite#217]
Does anybody know why there appears a red cross on white background when "unpinning" a docked window? Seems to be the DockPanel background which causes that graphics crash somehow. 
![beforehide](https://cloud.githubusercontent.com/assets/7810064/3424466/5b85a2b6-ffd1-11e3-9c7e-78f78f804994.jpg)
![afterhide](https://cloud.githubusercontent.com/assets/7810064/3424467/5ffd9d80-ffd1-11e3-81f7-cb67e30430b5.jpg)
That red cross appeared after a docked window (not visible in these screenshots) was "unpinned". It was possible to "re-pin" that window with a right mouse click into that red crossed area, again. 
Thanks for any hint in advance.

Kind regards

**Comment by bogunjemiyo** [comment]
I know this issue is closed. But I am writing to support your comment that the probable cause of the problem was the icon file used on the form that inherits from DockContent. I had the same exact issue with one of my forms that I docked to the left in a C# win form application. I removed the icon and used a different one and the issue was gone. In my case I believe the icon size was too big it was a *.ico file dimension 256 x 256 . Thank you..
